app/src/main/java/org/schabi/newpipe/settings/SettingMigrations.java

```diff
@@ -143,6 +157,33 @@ protected void migrate(@NonNull final Context context) {
         }
     };
 
+    private static final Migration MIGRATION_6_7 = new Migration(6, 7) {
+        @Override
+        protected void migrate(@NonNull final Context context) {
+            // The SoundCloud Top 50 Kiosk was removed in the extractor,
+            // so we remove the corresponding tab if it exists.
+            final TabsManager tabsManager = TabsManager.getManager(context);
+            final List<Tab> tabs = tabsManager.getTabs();
+            final List<Tab> cleanedTabs = tabs.stream()
+                    .filter(tab -> !(tab instanceof Tab.KioskTab kioskTab
+                            && kioskTab.getKioskServiceId() == SoundCloud.getServiceId()
+                            && kioskTab.getKioskId().equals("Top 50")))
+                    .collect(Collectors.toUnmodifiableList());
+            if (tabs.size() != cleanedTabs.size()) {
+                tabsManager.saveTabs(cleanedTabs);
+                // create an AlertDialog to inform the user about the change
+                // do not translate text, it is only shown once
+                MIGRATION_INFO.add((Context uiContext) -> new AlertDialog.Builder(uiContext)
+                            .setTitle("SoundCloud Top 50 page removed")
+                            .setMessage("SoundCloud has discontinued the original Top 50 charts. "
+                                    + "The corresponding tab has been removed from your main page.")
+                            .setPositiveButton(android.R.string.ok, null)
```

Comment:
I disagree, this should be translated like other related strings!

Please also use the OK string from NewPipe.